src/main/java/bowling/domain/Rolls.java

```diff
@@ -0,0 +1,39 @@
+package bowling.domain;
+
+import bowling.dto.RollsDto;
+
+import java.util.ArrayList;
+import java.util.stream.IntStream;
+
+import static java.util.stream.Collectors.collectingAndThen;
+import static java.util.stream.Collectors.toList;
+
+class Rolls {
+    // NOTE: get 에서의 bigO 를 1로 보장하기 위해, ArrayList 사용
+    private final ArrayList<Roll> rolls = new ArrayList<>();
```

Comment:
추후 변경이 용이하도록 타입은 좀 더 추상된 형태로 사용해보는 것은 어떨까요~? 
```suggestion
    private final List<Roll> rolls = new ArrayList<>();
```